examples/access-plugin-minimal/watcherjob.go

```diff
@@ -28,7 +28,16 @@ func (g *googleSheetsClient) HandleEvent(ctx context.Context, event types.Event)
 		return nil
 	}
 
-	r := event.Resource.(types.AccessRequest)
+	if _, ok := event.Resource.(types.WatchStatus); ok {
+		fmt.Println("Successfully started listening for Access Requests...")
```

Comment:
Instead of swallowing `WatchStatus` return, should we have the `GetKinds()` included in this print statement?

Reply:
We could but given this is an example, I don't think it's that important.
It adds a bit of complexity without any real benefit.

Well, no strong opinion but I would rather keep it as is.